ProjectSource/src/test/java/com/kayako/sdk/ParserFactoryTest.java

```diff
@@ -0,0 +1,222 @@
+package com.kayako.sdk;
+
+import com.kayako.sdk.base.parser.DataNodeParser;
+import com.kayako.sdk.error.response.*;
```

Comment:
⚠️  It could be a good improvement to list imports and avoid Type Import on Demand (`*`). The advantage of explicitly listing all imports is that another person (developer/reviewer) can tell at a glance which classes are meant to use, which simply makes reading the code that much easier. By a quick one-off thing, there's nothing explicitly wrong, but future maintainers will thank you for the clarity otherwise. Moreover, the code avoids cluttering the local namespace.  

**[More details](https://confluence.devfactory.com/x/gTCkEw)**
          
## The possible solution in this case

```Java  
import CLASS_1;
import CLASS_2;
...
```
 

<a id="a2538ef0-50c3-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />